lib/src/method_channel.dart

```diff
@@ -43,32 +46,45 @@ class ShowMenuArgs {
   }
 }
 
-const _channel = MethodChannel('native_context_menu');
+final _channel = const MethodChannel('native_context_menu')
+  ..setMethodCallHandler(
+    (call) async {
+      switch (call.method) {
+        case 'id':
```

Comment:
rename to `onItemSelected`
also it makes sense to add another method `onMenuDismissed` instead of sending `-1`